packages/shared/routes/dashboard/Dashboard.svelte

```diff
@@ -409,11 +409,18 @@
     $: if ($accountsLoaded) {
         setSelectedAccount($activeProfile.lastUsedAccountId ?? $viewableAccounts?.[0]?.id ?? null)
     }
+
+    $: if (!$activeProfile.hasFinishedSingleAccountGuide) {
+        openPopup({ type: 'singleAccountGuide', hideClose: true, overflow: true })
+    }
```

Comment:
This will clash with the popup online 214, we should move that crash reporting popup to when the app loads, if they have existing profiles but after they have agreed the new legal update popup if there is one.